app/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 100,
+  "parser": "typescript",
+  "tabWidth": 2
```

Comment:
Is there any way we can say indent with `spaces`?

Reply:
We can specify `"useTabs": false` for preventing indentation with tabs. But that is `false` by default. So no need. See [this](https://prettier.io/docs/en/options.html#tabs)